tests/test_transition_incremental.py

```diff
@@ -61,7 +66,7 @@ def test_transition_fs_sync_cow_full(self):
         self.assertIsNotNone(snapdev)
 
         self.assertEqual(snapdev["error"], -errno.EFBIG)
-        self.assertEqual(snapdev["state"], elastio_snap.State.ACTIVE | elastio_snap.State.SNAPSHOT)
```

Comment:
The driver can clear the SNAPSHOT bit, so we just check for ACTIVE. The presence of SNAPSHOT isn't important anymore, as the driver is in the fail state.